Source/JavaScriptCore/runtime/FileBasedFuzzerAgent.cpp

```diff
@@ -55,14 +55,9 @@ SpeculatedType FileBasedFuzzerAgent::getPredictionInternal(CodeBlock* codeBlock,
 
     // FIXME: the output of codeBlock->expressionRangeForBytecodeIndex() allows for some of
     // these opcodes to have predictions, but not all instances can be reliably targeted.
-    case op_bitand: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203604
-    case op_bitor: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203604
-    case op_bitxor: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203604
     case op_get_from_scope: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203603
     case op_get_from_arguments: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203608
     case op_get_by_val: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203665
-    case op_rshift: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203664
-    case op_lshift: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203664
```

Comment:
Ditto: looks like https://bugs.webkit.org/show_bug.cgi?id= 203664 is no longer relevant.  Let's close it as Config Changed.